algorithms/Steamroller.js

```diff
@@ -0,0 +1,20 @@
+// Flatten a nested array. You must account for varying levels of nesting.
+
+
+function steamrollArray(arr) {
+  // I'm a steamroller, baby
```

Comment:
I suggest omitting irrelevant comments in code that is ready for merging - comments have the potential to confuse the reader if they are not relevant, and also add a maintenance burden for future developers (as the code changes, comments related to that code need to be kept up to date as well).